packages/reports/addon/components/grouped-list.ts

```diff
@@ -17,9 +17,19 @@ interface Args {
 }
 
 export default class GroupedListComponent extends Component<Args> {
+  constructor() {
+    //@ts-ignore
+    super(...arguments);
+    if (Object.keys(this.groupedItems).length === 1) {
+      this.isSingleCategory = true;
+    }
+  }
+
```

Comment:
```suggestion
```
This is better done as a getter so it can be reactive to new `groupedItems` that are passed in.

Reply:
if I make this into a getter it will respond to the search function - is that what we want to happen?

Reply:
I think that should be alright, we just want to prevent unecessary clicks. We could probably add a getter for `shouldOpenAllGroups` to check if there are only a few (maybe 5?) items.

Could we also pick up #1338